integration-tests/config.ts

```diff
@@ -140,6 +141,15 @@ const oxfordnetEphemeral: Config =
 const oxfordnetSecretKey: Config =
   { ...oxfordnetEphemeral, ...{ signerConfig: defaultSecretKey }, ...{ defaultRpc: 'http://ecad-oxfordnet-full:8732' } };
 
+const nairobinetSecretKey: Config = 
+  defaultConfig({
+    networkName: 'NAIROBINET',
+    protocol: Protocols.PtNairobi,
+    defaultRpc: 'http://ecad-nairobinet-full:8732',
```

Comment:
defaultRpc: 'http://localhost:20000'

Reply:
it actually doesn't matter since it just acts like a placeholder, but I'll update it